scripts/index.sh

```diff
@@ -24,5 +24,7 @@ yq -sS 'map(
     "\(.publisher)/\(.name)/\(.version)" as $id |
     {
         $id, displayName, version, type, name, description, publisher,
-        links: {self: "\($PLUGINS_DIR)/\($id)"}
-    }) | sort_by(.id)' "${metas[@]}" --arg "PLUGINS_DIR" "$PLUGINS_DIR"
+        links: {self: "\($PLUGINS_DIR)/\($id)"},
+        deprecate
+    } | if .deprecate == null then del(.deprecate) else . end) |
```

Comment:
This can also be made a little cleaner I think:
```
yq -sS 'map(
    "\(.publisher)/\(.name)/\(.version)" as $id |
    {
        $id, displayName, version, type, name, description, publisher,
        links: {self: "\($PLUGINS_DIR)/\($id)"}
    } + if has("deprecate") then {deprecate} else null end ) | 
    sort_by(.id)' "${metas[@]}" --arg "PLUGINS_DIR" "$PLUGINS_DIR"
```

Reply:
Thanks! This is better way of adding deprecate field in the index json file.